### DISCUSSION TITLE
Constant subscription errors

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#17705] [Q&A]
We have Raven Cloud and a ton of subscriptions. We're constantly getting the following error from various subscriptions.

`[Subscription_SalesDataProcessing] for [client]- Connection retry Subscription With Id 'Subscription_SalesDataProcessing' cannot be processed by current node 'B', it will be redirected to ]\nReasons:`

Could I be setting up my subscriptions wrong? Is this something I need to concern myself with? I've had to modify my logging to not mark this as an error as it was clogging up the logs.

**Comment by garayx** [comment]
Hi

This is a transient error, which means that the responsible node for the subscription's task changed, and the subscription worker should reconnect to the new node, and continue processing.

**Reply by ItsRobbAllen** [reply]
Should it be as common as it seems to be? I could understand getting this once every blue moon, but it's 90% of our error log right now.

Which makes me think maybe I'm doing something wrong in my subscriptions.

**Reply by georgiosd** [reply]
We're also seeing this but haven't had time to quantify it. I don't know if it's easy for you to analyze the logs @ItsRobbAllen but it'd probably help the RDB guys tell us if it's expected.

**Reply by garayx** [reply]
if you get this error constantly, usually its an indication for unstable cluster, especially when the log reads `it will be redirected to ` (after `redirect to` is null) which means there is no member that can be the [responsible node](https://ravendb.net/docs/article-page/6.0/csharp/server/clustering/distribution/highly-available-tasks#responsible-node) and  [handle the subscription task](https://ravendb.net/docs/article-page/5.4/csharp/studio/database/settings/manage-database-group#members-duties-are) currently

**Reply by ItsRobbAllen** [reply]
We're using Cloud and man, we get this error so much it makes parsing through the logs difficult.

**Reply by georgiosd** [reply]
@ItsRobbAllen as in RDB cloud or self-hosted in another cloud provider?

I'll see if I can collect some stats on our side to compare.
